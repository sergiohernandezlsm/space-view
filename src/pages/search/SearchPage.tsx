import Container from "react-bootstrap/Container";
import Row from "react-bootstrap/Row";
import styles from "./SearchPage.module.scss";
import CardCounter from "../../components/card-counter";
import SearchForm from "../../components/search-form";

const SearchPage = () => {
  return (
    <Container className={styles.searchPageWrapper}>
      <Row>
        <h1>NASA Search</h1>
        <SearchForm />
      </Row>
      <Row>
        <CardCounter />
      </Row>
    </Container>
  );
};

export default SearchPage;
