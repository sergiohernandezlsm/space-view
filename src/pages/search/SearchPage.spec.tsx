import { shallow } from "enzyme";
import SearchPage from "./SearchPage";

describe("SearchPage", () => {
  it("should render SearchPage", () => {
    const wrapper = shallow(<SearchPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
