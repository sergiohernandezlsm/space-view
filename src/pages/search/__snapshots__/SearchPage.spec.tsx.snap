// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchPage should render SearchPage 1`] = `ShallowWrapper {}`;
